db/seed.js

```diff
@@ -1,27 +1,223 @@
+          'https://target.scene7.com/is/image/Target/51530146?wid=250&hei=250&qlt=80&fmt=webp',
+      },
+      {
+        name: 'Apple Lightning to Digital AV Adapter',
+        price: 49.99,
+        imageURL:
+          'https://target.scene7.com/is/image/Target/51677595?wid=250&hei=250&qlt=80&fmt=webp',
+      },
+    ],
+    non_Apple_Tablets: [
+      {
+        name: 'Targus New Stylus for Tablets/Other Touch Screen - Black (AMM165US)',
+        price: 8.99,
+        imageURL:
+          'https://target.scene7.com/is/image/Target/GUEST_cdfec344-3212-450a-8e51-556adef8ed7e?wid=253&hei=253&qlt=80&fmt=webp',
+      },
+      {
+        name: "Skullcandy Ink'd Wired Earbuds With Microphone",
+        price: 9.69,
+        imageURL:
+          'https://target.scene7.com/is/image/Target/GUEST_1be1b35c-ce83-4b4d-bb9f-de8a3ed36ed9?wid=253&hei=253&qlt=80&fmt=webp',
+      },
+      {
+        name: 'Case Logic Surefit Universal Tablet for Galaxy/Nexus and Kindle - Black',
+        price: 17.99,
+        imageURL:
+          'https://target.scene7.com/is/image/Target/GUEST_099f7a04-605b-4d0a-a540-31a7364fca82?wid=325&hei=325&qlt=80&fmt=webp',
+      },
+      {
+        name: 'Beats X Wireless Earphones - Silver',
+        price: 119.99,
+        imageURL:
+          'https://target.scene7.com/is/image/Target/GUEST_1be1b35c-ce83-4b4d-bb9f-de8a3ed36ed9?wid=253&hei=253&qlt=80&fmt=webp',
+      },
+      {
+        name: 'Atomi - Mini Surge Protector',
+        price: 14.99,
+        imageURL:
+          'https://target.scene7.com/is/image/Target/GUEST_47d6abe9-e38d-4f49-a8f7-04f24dd3b7ac?wid=325&hei=325&qlt=80&fmt=webp',
+      },
+    ],
+    random: [],
+  };
 
-    ORM.Accessories.create({
-      name: 'workeddd',
-      price: 59.99,
-      imageURL: '1513eahaeh53151',
+  for (let i = 1; i <= 5; i++) {
+    accessories.random.push({
+      name: faker.commerce.productName(),
+      price: faker.finance.amount(),
+      imageURL: faker.random.image(),
     });
+  }
 
-    ORM.ViewHistory.create({
-      productID: 1,
+  return accessories;
+}());
+
+db.sync({ force: true })
+  .then(() => {
+    categoriesData.forEach((categoryItem) => {
+      Categories.create({
+        name: categoryItem,
+      })
+        .then((category) => {
+          const categoryName = category.get('name');
+
+          productsData[categoryName].forEach((product) => {
+            Products.create({
+              name: product.name,
+              price: product.price,
+              imageURL: product.imageURL,
+              categoryName: category.get('name'),
+            });
+          });
+
+          accessoriesData[categoryName].forEach((accessory) => {
+            Accessories.create({
+              name: accessory.name,
+              price: accessory.price,
+              imageURL: accessory.imageURL,
+              categoryName: category.get('name'),
+            });
+          });
+        })
+        .then(() => console.log('worked!'));
```

Comment:
this `'worked!'` console.log will appear whether or not each of the `Products.create(...)` or `Accessories.create(...)` successfully occurred or not